### PR TITLE
make forcefield controlabe with digilines

### PR DESCRIPTION
**Pull request by DS-Minetest**
Commands are `get`, `range <int>`, `off`, `on`, `toggle`, `shape <0/1>` and everyone again with capital letters.
related: #168

### CODE REVIEW

File: technic/machines/HV/forcefield.lua [hunk 1]
```diff
@@ -150,6 +160,40 @@ local mesecons = {
...
+					range   = meta:get_int("range"),
+					shape   = meta:get_int("shape")
+				})
+			elseif msg == "off" then
+				meta:set_int("enabled", 0)
+			elseif msg == "on" then
+				meta:set_int("enabled", 1)
+			elseif msg == "toggle" then
+				local onn = meta:get_int("enabled")
+				onn = -(onn-1) -- switch 0 to 1 and vice versa
```
Comment (DS-Minetest) [inline technic/machines/HV/forcefield.lua]: `onn = 1-onn`

**Comment by DS-Minetest** [comment]
Everything is ok now IMO. Could please someone review?

**Comment by kaeza** [comment]
https://github.com/minetest-technic/technic/commit/ef8bb38bfd8e52a511fbe930257b173ddab93f63

Thanks!

**Comment by numberZero** [comment]
@DS-Minetest thank for your work, but

> `range <int>`

please don’t use such homegrown serialization formats. Digilines do support tables, you know. Only simplest commands, like `get`, may be sent as plain strings.

Also... what if `msg` is not a string? `msg:lower()` will crash the server then, I suppose.

**Comment by DS-Minetest** [comment]
I made the range thing to string so, that you can easily type it by hand. Both could be allowed.

For the type of msg should really be checked. Perhaps I'll open a new PR to improve this..

**Pull request closed without merging**